sklearn/ensemble/_hist_gradient_boosting/gradient_boosting.py

```diff
@@ -1164,6 +1164,18 @@ class HistGradientBoostingRegressor(RegressorMixin, BaseHistGradientBoosting):
 
         .. versionadded:: 0.24
 
+    See Also
+    --------
+    GradientBoostingRegressor : Exact gradient boosting method that does not
+        scale as good on datasets with a large number of samples.
+    DecisionTreeRegressor : A decision tree regressor.
+    RandomForestRegressor : A random forest regressor.
```

Comment:
In the other PR, you added more information that we could use as well here. To recall:

```
A meta-estimator that fits a number of decision tree classifiers on various sub-samples of the dataset and uses averaging to improve the predictive accuracy and control over-fitting.
```

Reply:
Yes, I didn't add it here because upon reading [RandomForestRegressor](https://scikit-learn.org/stable/modules/generated/sklearn.ensemble.RandomForestRegressor.html), I questioned if the description is correct. In particular this part "A random forest is a meta estimator that fits a number of __classifying__ decision trees ...". Maybe it is right but confusing (at least for me). Wouldn't it be better to put something like "A random forest is a meta estimator that fits a number of decision tree __regressors__ ...". What do you think?

Reply:
```
A meta-estimator that fits a number of decision tree regressors on various sub-samples of the dataset and uses averaging to improve the statistical performance and control over-fitting.

```